tests/console/check_migration_features.pm

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Package: SUSEConnect
+# Summary: Verify migration features on target system.
+# Maintainer: Lemon Li <leli@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use Utils::Architectures;
+use utils;
+use version_utils;
+use registration;
+use List::MoreUtils 'uniq';
```

Comment:
use registration;
use List::MoreUtils 'uniq';

these 2 modules maybe not used here

Reply:
Updated.